modules/ROOT/pages/changelogs.adoc

```diff
@@ -26,6 +26,25 @@
 //Starting from 2025.01, when using Cypher25, queries using `WAIT` return the notifications listed above instead of result rows.
 //The behaviour for Cypher5 remains unchanged.
 
+== Neo4j 5.26
+
+**New:**
+
+[options="header", cols="<1m,<1"]
+|===
+
+| GQLSTATUS
+| Neo4j code
+
+| 01N72
+| Neo.ClientNotification.Statement.InsecureProtocol
+
+| 42I50
+| Neo.ClientError.Schema.TokenLengthError
```

Comment:
This one is not documented, needs to be added to the "List of Neo4j error codes" page also.